stable/stash/templates/deployment.yaml

```diff
@@ -41,6 +41,7 @@ spec:
         - --audit-log-path=-
         - --tls-cert-file=/var/serving-cert/tls.crt
         - --tls-private-key-file=/var/serving-cert/tls.key
+        - --enable-analytics={{ .Values.enableAnalytics }}
```

Comment:
Enabling this would require a bit more heads-up! Make sure there's a note in the README about it and a description of the enableAnalytics variable. This would also warrant a bump of the minor version of this Chart, as it adds a new feature. I'd prefer to see this handled in a separate PR.

Reply:
Done.